pkgs/applications/science/electronics/bitscope/packages.nix

```diff
@@ -0,0 +1,216 @@
+{ stdenv, fetchurl, buildFHSUserEnv, makeWrapper, pkgs }:
+
+let
+  wrapBinary = libPaths: binaryName: ''
+    wrapProgram "$out/bin/${binaryName}" \
+      --prefix LD_LIBRARY_PATH : "${stdenv.lib.makeLibraryPath libPaths}"
+  '';
+
+  mkBitscope = attrs: let
+    mkBitscopeTool = attrs: stdenv.mkDerivation (rec {
+      inherit (attrs) name version src;
+
+      meta = with stdenv.lib; {
+        homepage = http://bitscope.com/software/;
+        license = licenses.unfree;
+        platforms = [ "i686-linux" "x86_64-linux" ];
+        maintainers = with maintainers; [
+        ];
+      } // (attrs.meta or {});
+
+      buildInputs = with pkgs; [
+        dpkg
+        makeWrapper
+      ];
+
+      libs = attrs.libs or (with pkgs; [
+        atk
+        cairo
+        gdk_pixbuf
+        glib
+        gtk2-x11
+        pango
+        xorg.libX11
+      ]);
+
+      dontBuild = true;
+
+      unpackPhase = attrs.unpackPhase or ''
+        dpkg-deb -x ${attrs.src} ./
+      '';
+
+      installPhase = attrs.installPhase or ''
+        mkdir -p "$out/bin"
+        cp -a usr/* "$out/"
+        ${(wrapBinary libs) attrs.toolName}
+      '';
+    });
+    pkg = mkBitscopeTool attrs;
+  in buildFHSUserEnv {
+    name = attrs.toolName;
+    meta = pkg.meta;
+    runScript = "${pkg.outPath}/bin/${attrs.toolName}";
+  };
+in {
+  dso = let
+    name = "${toolName}_${version}";
```

Comment:
Tip: If package name and version is separated with a dash, Nix will know how to parse it. This currently uses underscore, so Nix will not know the version.